p4-14/v1.0.5/tex/p4.pt

```diff
@@ -2160,21 +2161,20 @@ continues as per the current control function specification.
 
 \primact{push(array, count)}
 { % Summary
-Push all header instances in an array down and add a new header at the top.
+Push all header instances in an array down and add a new invalid header at the top.
```

Comment:
One reasonable alternative is to continue to define push() in P4_14 programs to add a new valid header at the top, i.e. to continue to allow this behavior to be different than P4_16's definition of the push_front operation.

There may be extant P4_14 programs that are relying on push() having the effect of making the new inserted headers valid, e.g. I think switch.p4 relies upon this.